examples/react-graph-gl/index.jsx

```diff
@@ -1,8 +1,9 @@
 import React, {Component} from 'react';
 
-import GraphGL, {D3ForceLayout, JSONLoader, NODE_TYPE} from 'modules/react-graph-layers';
+import {D3ForceLayout, JSONLoader, NODE_TYPE} from 'deck-graph-layers';
+import {GraphGL} from 'react-deck-graph-layers';
 
-import SAMPLE_GRAPH_DATASETS from '../../utils/data/sample-datasets';
+import SAMPLE_GRAPH_DATASETS from 'react-deck-graph-layers/test/utils/data/sample-datasets.js';
```

Comment:
Yeah, I'd prefer to avoid the deep import too. Worried about adding an export though, unsure what bundlers will do if the imported data is missing from the npm package, and probably we don't want it published there. 

Maybe a new module `modules/test-utils` with shared data and utils for tests? Or perhaps a shared alias in a config file?

Reply:
Yes... FWIW I personally think all examples / all code in our repos should be `.ts/.tsx`. If we do that and want users to be able to copy these folders and run on their own, we may need the dependency in each example folder. That is how I do it in luma, loaders etc.

Reply:
I've mode the dataset to `@deck.gl-community/test-utils` for now, and switched that and the template module to TypeScript + Vitest. I'm not sure if "may need the dependency in each example folder" means an npm dependency or just copying the data? For examples and tests I think having it be very obvious what the data looks like can be helpful, maybe an inline dataset would eventually be the choice here, but that's for another PR I think.

Reply:
Sounds good. I would vote for having the datasets for the examples in the example folder. They should be standalone as far as possible

Having datasets from all different modules in the test-utils seems to centralized.